ui/tract-renderer/src/main/kotlin/org/cru/godtools/tract/activity/TractActivity.kt

```diff
@@ -109,12 +109,9 @@ class TractActivity :
     }
 
     override fun onCreateOptionsMenu(menu: Menu) = super.onCreateOptionsMenu(menu).also {
-        menuInflater.inflate(R.menu.activity_tract, menu)
```

Comment:
by removing this you have removed the install menu item that is used in the instant app

Reply:
oh, you ended up reordering it and adding it below